app/containers/User/index.js

```diff
@@ -64,7 +65,9 @@ export class User extends React.Component {
           </h1>
           { user.type === 'studs_member' ?
```

Comment:
I prefer
`user.type === 'studs_member' && <...>`
over writing
`bool ? <...> : null`

Reply:
Yep. I change this in #37 :)